CommandDotNet.Tests/Utils/TestFactory.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using System.Reflection;
+using CommandDotNet.Models;
+
+namespace CommandDotNet.Tests.Utils
+{
+    public static class TestFactory
+    {
+        private static readonly AppSettings DefaultAppSettings = new AppSettings();
```

Comment:
in time, we may determine it's better to create an instance of TestFactory w/ AppSettings so all methods will use the same one.  I didn't yet because I'm not sure how frequently we'll use this.